fastlane/Fastfile

```diff
@@ -11,6 +11,102 @@ before_all do
     setup_ci()
   end
 end
+desc "Release a new version"
+lane :release do |options|
```

Comment:
With this lane, Fastfile is becoming bigger and bigger. Honestly I don't like the Fastfile containing everything on one pile. Maybe we can split this into separate files? Should I create ticket for that? @b-onc